concordium-std/src/test_infrastructure.rs

```diff
@@ -1326,17 +1337,18 @@ impl<State: Serial + DeserialWithState<TestStateApi> + StateClone<TestStateApi>>
         if self.missing_accounts.contains(receiver) {
             return Err(TransferError::MissingAccount);
         }
-        if amount.micro_ccd > 0 {
```

Comment:
Let's not change behaviour we have no strong reason for changing. Just because a change is documented does not mean people are not annoyed when there is not much of a reason for it.

Reply:
Holm correctly realized that I was wrong in my comment. My statement was true back when the contract balance was an `Option<Amount>` in the context, but now it is a `RefCell<Amount>` in the host.